packages/Switch/stories/Switch.backyard.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { getStoryName } from "storybook/storyTree";
+import { testStoryParameters } from "storybook/assets/storyParameters";
+import Switch from "../src/Switch";
+import Screener from "./tests/Screener";
```

Comment:
I'd suggest just `import Screener from "./examples/Variations":` here, then you don't even need a `Screener.js` file at all! 🧹